flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why intel only?

Reply:
arm may compile but is untested and/or broken, usually because the game currently uses gl3.1 only and has no functioning support for gles

Reply:
it clear that nobody can test it if you never build it. and there are aarch64 platform with full GL.

Reply:
yeah nvm, does work. i'll add it to build ci upstream so it stays compiling at least